undebt/cmd/main.py

```diff
@@ -57,21 +57,21 @@ def _write_result_text(result_text, path, dry_run):
 def _handle_arguments():
     parser = argparse.ArgumentParser(prog='undebt')
     parser.add_argument(
-        '--input', '-i', metavar='path', action='append',
```

Comment:
I think it's worth keeping around `-i` for backwards compatibility reasons, and just use both the positional arguments and the `-i` arguments to build the full list of inputs.